NEWS.md

```diff
@@ -1,3 +1,7 @@
+## cyphr 1.0.6
+
+* Added wrappers for `readxl::read_excel` and `writexl::write_excel`
```

Comment:
Don't know how much you care about this but this doesn't match up with the functions you register here?

Reply:
looks like I got the rebase wrong

Reply:
oh no, that's right